src/KBS.FauxApplication/KBS.FauxApplication.csproj

```diff
@@ -5,4 +5,13 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="MassTransit" Version="5.1.5" />
+    <PackageReference Include="MassTransit.RabbitMQ" Version="5.1.5" />
```

Comment:
The InMemory is the only transport that is used for now, that means that this package isn't needed.